XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+    Amount: "10000000",
+    CredentialIDs: ["DD40031C6C21164E7673...17D467CEEE9"]
+}
+```
+
+## 13. Security
+
+### 13.1. Trust Assumptions
+
+You need to trust the issuer to only be issuing valid credentials, and to not delete a credential without a legitimate reason.
+
+### 13.2. Data Privacy
+
+No private data needs to be stored on-chain (e.g. all the KYC data). The actual VC can still be stored off-chain. It could be stored at a link in the `URI` field.
+
+# Appendix
+
+## Appendix A: FAQ
+
+### A.1: Do I have to only use credentials with the Deposit Authorization feature? Can I also use them for off-chain purposes?
+
+You can use credentials for any purpose you'd like, not just for Deposit Authorization.
+
+### A.2: Do I need to use credentials for VCs or can I use them for other things?
+
+You can use credentials for any purpose you'd like. The recommended usecase is VCs, but there are plenty of other usecases for them.
+
+### A.3: Why not use NFTs instead, with a certain issuer-taxon combo and the `tfTransferable` flag unset?
+
+Using NFTs instead wouldn't be as elegant, since it means that the NFT has to follow a specific format. It would make more sense for an off-ledger use-case, not directly on-ledger. In addition, an NFT serves different needs - e.g. NFTs don't have an expiration.
+
+### A.4: Can an issuer issue a credential for themselves?
+
+Yes.
+
+### A.5: Can an issuer edit the details of a credential once it's created?
+
+No, it would have to be deleted and recreated. This is similar to needing to get a new card if your license/passport expires. If the `Subject`, `Issuer`, and `CredentialType` stay the same, though, the object will still have the same ID as before.
+
+### A.6: Why do I need to include the `CredentialIDs` field in my transaction? Shouldn't the XRPL be able to figure out whether I have a valid credential automatically?
+
+There is no way for the XRPL to iterate through just the list of credentials an account has; it can only iterate through the entire list of account objects. This is an unbounded list of objects (it could be millions of objects). Even just the list of accepted credentials could theoretically be millions of objects long. It could cause serious performance issues if the system had to iterate through the whole list.
+
+It's much faster to have the credential ID included - it's easy to make sure that that's a valid credential, and check that it's an authorized credential.
+
+### A.7: Can a credential issuer delete their account?
+
+Yes, even if they still have issued credentials in existence.
+
+### A.8: Does a credential issue have to have an on-chain account?
+
+No, though they must have a valid keypair. If the second method of creating a credential is used (via a signature from the issuer), the issuer can still attest to a credential without having to have an account on-chain. This is to ensure closer compliance with the W3C Verifiable Credentials spec (see section 9).
```

Comment:
I don't think this holds with the spec for `CredentialCreate` as currently defined since it states that the transaction should fail if `Issuer` specifies an account that does not exist, but you have to specify that field if you're the subject submitting a credential that was signed off-chain.

Reply:
That's leftover from a previous version of the spec (which did not include the second method). That has been amended.